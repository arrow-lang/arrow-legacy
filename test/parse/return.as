return 40;
return -40;
return not false or not true;
return 430 + 302;
