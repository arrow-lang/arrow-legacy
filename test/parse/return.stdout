<?xml version="1.0" encoding="utf-8"?>
<Module>
 <Return>
  <Integer base="10">40</Integer>
 </Return>
 <Return>
  <NegateNumeric>
   <Integer base="10">40</Integer>
  </NegateNumeric>
 </Return>
 <Return>
  <Or>
   <NegateLogical>
    <Boolean>false</Boolean>
   </NegateLogical>
   <NegateLogical>
    <Boolean>true</Boolean>
   </NegateLogical>
  </Or>
 </Return>
 <Return>
  <Add>
   <Integer base="10">430</Integer>
   <Integer base="10">302</Integer>
  </Add>
 </Return>
</Module>
